### DISCUSSION TITLE
lab4-0 problem

### DISCUSSION THREAD

**Discussion by AsWhiteAsTofu** [bol-edu/HLS-SOC-Discussions#140] [Q&A]
想請問一下，在執行 Generate Caravel SoC FPGA Bitstream from Xilinx Vivado 的 Xilinx Vivado Synthesis 步驟時，執行`$ ./run_vivado`會顯示`ERROR: [Common 17-69] Command failed: Run 'impl_1' has not been launched. Unable to open`，想請問該如何解決 謝謝
![messageImage_1699452215146](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145284535/f27ad77d-00a3-4875-a34b-1301582f9281)
![messageImage_1699383857888](https://github.com/bol-edu/HLS-SOC-Discussions/assets/145284535/6310a9d5-2ff5-4b60-b04c-7bf92becbcda)

**Comment by jimmy20000314** [comment]
一般來說流程是synthesis -> implementation -> generate bitstream,
這邊看來是synthesis fail了, 所以implementation一直等不到前面的檔案，才會說impl_1尚未執行
